src/ui/base-page-login.js

```diff
@@ -0,0 +1,39 @@
+/*
+For copyright information, see the AUTHORS.md file in the docs directory of this distribution and at
+https://github.com/fluid-project/sjrk-story-telling/blob/main/docs/AUTHORS.md
+
+Licensed under the New BSD license. You may not use this file except in compliance with this licence.
+You may obtain a copy of the BSD License at
+https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/main/LICENSE.txt
+*/
+
+"use strict";
+
+(function ($, fluid) {
+
+    // The login page base grade
+    fluid.defaults("sjrk.storyTelling.base.page.login", {
+        gradeNames: ["sjrk.storyTelling.base.page.authentication"],
+        pageSetup: {
+            authenticationUrl: "/login"
+        },
+        components: {
+            authenticationUi: {
+                options: {
+                    components: {
+                        templateManager: {
+                            options: {
+                                templateConfig: {
+                                    // SJRK-340: this would be a perfect example
+                                    // of why this Jira exists
+                                    templatePath: "%resourcePrefix/templates/login.hbs"
```

Comment:
Could just be an options distribution, of course - 
````
    distributeOptions: {
        "login.templatePath": {
            record: "%resourcePrefix/templates/login.hbs",
            target: "{that ui templateManager}.options.templateConfig.templatePath"
        }
    }
````

I notice that you have some of these already in "sjrk.storyTelling.base.page", as well as several targetting this exact option in the themes

Reply:
An options distribution would work, though it still assumes knowledge of the `templateManager`'s inner workings. I filed [SJRK-340](https://issues.fluidproject.org/browse/SJRK-340) to handle this improvement in a way that would likely use such an options distribution, but wouldn't necessarily require knowledge of it, nor would it require specifying a fresh one for every `ui` grade.